packages/google_sign_in/google_sign_in_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.3.0
+
+* Adopts `plugin_platform_interface`.
+  * Deprecates `isMock` in favor of the now-standard
```

Comment:
```suggestion
* Deprecates `isMock` in favor of the now-standard
```

Reply:
Why remove the nesting? The deprecation of `isMock` is because of the adoption of `plugin_platform_interface` (which uses the mixin); that's why I nested it.

Reply:
Not a huge deal but if you wanted to communicate some linkage between those items it would be more clear to do so with prose or to have some label for the sublist.

Reply:
Collapsed into a single bullet point.